### PR TITLE
BAH-2763 | configuration for sharing prescriptions via sms and email

### PR DESCRIPTION
**Pull request by kavitha-sundararajan**

### CODE REVIEW

File: masterdata/configuration/globalproperties/gp_clinical.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<config>
+  <globalProperties>
+    <globalProperty>
+      <property>bahmni.enableSMSPrescriptionOption</property>
```
Comment (angshu) [inline masterdata/configuration/globalproperties/gp_clinical.xml:4]: is the property defined through any OMOD? if not, then you must do so through the OMOD's config.xml .. you can put a default value and then through the config (initializer) add this.

File: openmrs/apps/clinical/app.json [hunk 2]
```diff
@@ -27,6 +27,7 @@
             "additionalPatientIdentifiers": [
                 "ABHA Address"
             ]
-        }
+        },
+        "smsLanguage": "SMS_LANGUAGE"
```
Comment (angshu) [inline openmrs/apps/clinical/app.json]: what is this property? it looks like a key

File: masterdata/configuration/globalproperties/gp_clinical.xml (hunk 1 above)
Comment (kavitha-sundararajan) [reply masterdata/configuration/globalproperties/gp_clinical.xml:4]: added the property through OMOD's config with default value

File: openmrs/apps/clinical/app.json (hunk 2 above)
Comment (kavitha-sundararajan) [reply openmrs/apps/clinical/app.json]: Yes, it is a key. removed it from the config

**Review by angshu — APPROVED** [review]

**Pull request closed without merging**